scripts/add-config-to-dato/dato.mjs

```diff
@@ -0,0 +1,80 @@
+import { SiteClient } from 'datocms-client'
+import { config } from 'dotenv-safe'
+config()
+const client = new SiteClient(process.env.DATO_API_TOKEN_FULL_ACCESS)
+const TAG_TYPE = '1518135'
```

Comment:
Is it a `TYPE`? Seems more clear to use `ID`? Maybe add a comment where these numbers are coming from?